src/Data/List/Membership/Propositional/Properties.agda

```diff
@@ -96,14 +100,14 @@ map-mapWith∈ = Membership.map-mapWith∈ (≡.setoid _)
 
 module _ (f : A → B) where
 
-  ∈-map⁺ : ∀ {x xs} → x ∈ xs → f x ∈ map f xs
+  ∈-map⁺ : x ∈ xs → f x ∈ map f xs
   ∈-map⁺ = Membership.∈-map⁺ (≡.setoid A) (≡.setoid B) (cong f)
 
-  ∈-map⁻ : ∀ {y xs} → y ∈ map f xs → ∃ λ x → x ∈ xs × y ≡ f x
+  ∈-map⁻ : y ∈ map f xs → ∃ λ x → x ∈ xs × y ≡ f x
   ∈-map⁻ = Membership.∈-map⁻ (≡.setoid A) (≡.setoid B)
 
-  map-∈↔ : ∀ {y xs} → (∃ λ x → x ∈ xs × y ≡ f x) ↔ y ∈ map f xs
-  map-∈↔ {y} {xs} =
+  map-∈↔ : (∃ λ x → x ∈ xs × y ≡ f x) ↔ y ∈ map f xs
+  map-∈↔ {xs}{y} =
```

Comment:
I like the refactoring to use `private` variables, but here it does mean that the implicit argument changes, which is technically a `breaking` change, but I think of a kind that we have elsewhere accepted without worrying too much.

Reply:
Indeed it seems extremely tedious to worry about backwards compatibility w.r.t. (hidden) variable argument order.

I would urge you to only worry about backw.compat. only modulo arg-order, especially given that Agda does not provide any flexibility for that (e.g. topological/dependency order instead of the current strict linear order).